aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -56,18 +58,49 @@ public static DescribeDbClustersRequest describeDbClustersRequest(final Resource
                 .build();
     }
 
+    public static DescribeDbClustersRequest describeDbClusterByIdentifierRequest(final String dbClusterIdentifier) {
```

Comment:
NIT: I believe the `ByIndentifierRequest` portion of the signature is made redundant by the parameter list. This however is mostly of preference nature and we can keep the method as it is :)